src/command_modules/azure-cli-netappfiles/azure/cli/command_modules/netappfiles/__init__.py

```diff
@@ -0,0 +1,31 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+from azure.cli.command_modules.netappfiles._help import helps # pylint: disable=unused-import
+
+
+class NetAppFilesCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        netappfiles_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.netappfiles.custom#{}')
+        super(NetAppFilesCommandsLoader, self).__init__(cli_ctx=cli_ctx,
+                                                min_profile='2017-03-10-profile',
+                                                custom_command_type=netappfiles_custom)
```

Comment:
You need to add logic to suppress the extension here or the older extension will override your newer module.

Reply:
> You need to add logic to suppress the extension here or the older extension will override your newer module.

Ok that makes good sense. What needs added to perform this?